encord/constants/enums.py

```diff
@@ -6,6 +6,7 @@ class DataType(StringEnum):
     IMG_GROUP = "img_group"
     DICOM = "dicom"
     IMAGE = "image"
+    DICOM_STUDY = "dicom_study"
```

Comment:
I think at this point we should also introduce forward compatibility similar to what we do [here](http://github.com/encord-team/encord-client-python/blob/0a33ba42f791df4ce4b71e483236adc1dc040475/encord/orm/label_row.py#L233-L235)